### PR TITLE
Sample migration 6

### PR DESCRIPTION
**Pull request by kul3r4**

### CODE REVIEW

File: src/20_Components/amp-dynamic-css-classes.html [hunk 1]
```diff
@@ -32,17 +35,15 @@
       display: none;
     }
     p {
```
Comment (sebastianbenz) [inline src/20_Components/amp-dynamic-css-classes.html]: can you scope this to only apply to this sample

File: src/20_Components/amp-dynamic-css-classes.html [hunk 2]
```diff
@@ -32,17 +35,15 @@
...
   </style>
 </head>
 <body>
 
-  <!-- You do not need to include a call to the amp-dynamic-css-classes component to initialize the classes on the body element -->
-
   <!-- #### Referrer -->
   <!--
-    Using these referrer classes can allow you to conditionally include elements.
+    Using these referrer classes can allow you to conditionally include elements. You do not need to include a call to the amp-dynamic-css-classes component to initialize the classes on the body element.
```
Comment (sebastianbenz) [inline src/20_Components/amp-dynamic-css-classes.html]: surround `amp-dynamic-css-classes` with `

File: src/20_Components/amp-dynamic-css-classes.html [hunk 3]
```diff
@@ -19,6 +19,9 @@
     For the purpose of this sample, we're creating some helper CSS classes that use the CSS rules defined by amp-dynamic-css-classes.  These allow us to more succinctly refer to the conditions in our code below.
```
Comment (sebastianbenz) [inline src/20_Components/amp-dynamic-css-classes.html]: Surround amp-dynamic-css-classes with `

File: src/20_Components/amp-dynamic-css-classes.html (hunk 1 above)
Comment (sebastianbenz) [reply src/20_Components/amp-dynamic-css-classes.html]: Suggestion: use a `div` instead and add a class, e.g. `dynamic-css`

File: src/20_Components/amp-experiment.html [hunk 4]
```diff
@@ -39,7 +38,7 @@
 </head>
 <body>
 
-  <!-- ## Basic Usage -->
+  <!-- ## Basic usage -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-experiment.html]: Can you also fix title case for `Development Tip`

File: src/20_Components/amp-dynamic-css-classes.html (hunk 1 above)
Comment (kul3r4) [reply src/20_Components/amp-dynamic-css-classes.html]: done

File: src/20_Components/amp-dynamic-css-classes.html (hunk 2 above)
Comment (kul3r4) [reply src/20_Components/amp-dynamic-css-classes.html]: done

File: src/20_Components/amp-dynamic-css-classes.html (hunk 3 above)
Comment (kul3r4) [reply src/20_Components/amp-dynamic-css-classes.html]: done

File: src/20_Components/amp-experiment.html (hunk 4 above)
Comment (kul3r4) [reply src/20_Components/amp-experiment.html]: done

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Pull request merged**